templates/tanstack-shell/app/components/LoginForm.tsx

```
@@ -0,0 +1,161 @@
+        await login({ data: value })
+        router.invalidate()
+        
+        if (onSuccess) {
+          onSuccess()
+        } else {
+          const targetPath = redirectTo || '/dashboard'
+          await router.navigate({ to: targetPath })
+        }
+      } catch (error) {
+        // Form will handle the error display
+        throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)